src/components/m-button/mixins/_button.scss

```diff
@@ -11,6 +11,11 @@
 
 // Buttons as specified at
 // https://design.axa.com/web-guidelines/buttons-and-links
+$color-button-bg--disabled: get-color($button-color, default, bg--disabled);
```

Comment:
Please move these variable declaration to it's relevant local scope.
All global `$variable`s sit in `../settings/button.scss`